enrolment-service/enrol/core/views.py

```diff
@@ -191,39 +187,62 @@ def retrieve(self, request, pk=None):
         return Response(status=status.HTTP_404_NOT_FOUND)
 
     def perform_update(self, serializer):
+        user = self.request.user
+        project = self.get_object()
+        previous_status = project.status
```

Comment:
This helps checking what the current state flow intends, so we can send the right email notifications (e.g. newly accepted project, vs reopened... to be implemented in a different ticket)